packages/@react-spectrum/s2/src/Table.tsx

```diff
@@ -11,85 +11,1089 @@
+import {Rect} from '@react-stately/virtualizer';
+import SortDownArrow from '../s2wf-icons/S2_Icon_SortDown_20_N.svg';
+import SortUpArrow from '../s2wf-icons/S2_Icon_SortUp_20_N.svg';
+import {useIsMobileDevice} from './utils';
+import {useLoadMore} from '@react-aria/utils';
+import {useLocalizedStringFormatter} from '@react-aria/i18n';
+
+interface S2TableProps {
+  /** Whether the Table should be displayed with a quiet style. */
+  isQuiet?: boolean,
+  /**
+   * Sets the amount of vertical padding within each cell.
+   * @default 'regular'
+   */
+  density?: 'compact' | 'spacious' | 'regular',
+  /**
+   * Sets the overflow behavior for the cell contents.
+   * @default 'truncate'
+   */
+  overflowMode?: 'wrap' | 'truncate',
+  /** How selection should be displayed. */
+  selectionStyle?: 'checkbox' | 'highlight',
+  // TODO: will we contine with onAction or rename to onRowAction like it is in RAC?
+  /** Handler that is called when a user performs an action on a row. */
+  onAction?: (key: Key) => void,
+  /**
+   * Handler that is called when a user starts a column resize.
+   */
+  onResizeStart?: (widths: Map<Key, ColumnSize>) => void,
+  /**
+   * Handler that is called when a user performs a column resize.
+   * Can be used with the width property on columns to put the column widths into
+   * a controlled state.
+   */
+  onResize?: (widths: Map<Key, ColumnSize>) => void,
+  /**
+   * Handler that is called after a user performs a column resize.
+   * Can be used to store the widths of columns for another future session.
+   */
+  onResizeEnd?: (widths: Map<Key, ColumnSize>) => void,
+  /** The current loading state of the table. */
+  loadingState?: LoadingState,
+  /** Handler that is called when more items should be loaded, e.g. while scrolling near the bottom. */
+  onLoadMore?: () => any
+}
+
+// TODO: Note that loadMore and loadingState are now on the Table instead of on the TableBody
+export interface TableProps extends Omit<RACTableProps, 'style' | 'disabledBehavior' | 'className' | 'onRowAction' | 'selectionBehavior' | 'onScroll' | 'onCellAction' | 'dragAndDropHooks'>, StyleProps, S2TableProps {
+}
+
+let InternalTableContext = createContext<TableProps & {scale?: Scale, layout?: S2TableLayout<unknown>, setIsInResizeMode?:(val: boolean) => void, isInResizeMode?: boolean}>({});
+
+const tableWrapper = style({
+  width: 'full'
+});
+
+const table = style<TableRenderProps & S2TableProps>({
+  height: 'full',
+  userSelect: 'none',
+  minHeight: 0,
+  minWidth: 0,
+  fontFamily: 'sans',
+  fontWeight: 'normal',
+  overflow: 'auto',
+  backgroundColor: {
+    default: 'gray-25',
+    isQuiet: 'transparent',
+    forcedColors: 'Background'
+  },
+  outlineColor: {
+    default: 'gray-300',
+    isFocusVisible: 'focus-ring',
+    forcedColors: 'ButtonBorder'
+  },
+  outlineWidth: {
+    default: 1,
+    isQuiet: 0,
+    isFocusVisible: 2
+  },
+  outlineStyle: 'solid',
+  borderRadius: {
+    default: size(6),
```

Comment:
the border radius for the table really doesn't fall into their scale? seems like a mistake in spectrum

Reply:
good call out, I'll left a question in the token figma

Reply:
Got a reply, the 6 is correct here. It was a middle ground decision where 4px felt too much like S1 and 8px felt too rounded.

Reply:
sigh